docs/scripts/gen-faq.js

```diff
@@ -0,0 +1,174 @@
+// read.js
+const fs = require('fs')
+const yaml = require('js-yaml')
+const { Remarkable } = require('remarkable')
+const yamlPath = './docs/faq.yaml'
+
+try {
+  // Generating FAQ.mdx
+
+  if (!fs.existsSync(yamlPath)) {
+    //file exists
+    console.warn('.yaml File does not exists, skipping generating FAQ')
+    return 0
+  }
+
+  let fayYaml = fs.readFileSync('yamlPath', 'utf8')
```

Comment:
This is probably meant to be a variable:

```suggestion
  let fayYaml = fs.readFileSync(yamlPath, 'utf8')
```